tests/Integration/inc/Engine/License/Subscriber/displayUpgradeSection.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\Engine\License\Subscriber;
+
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\Engine\License\Subscriber::display_upgrade_section
+ *
+ * @group License
+ * @group AdminOnly
+ */
+class DisplayUpgradeSection extends TestCase {
+	private static $user;
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		$container  = apply_filters( 'rocket_container', null );
+		self::$user = $container->get( 'user' );
+	}
+
```

Comment:
Same here -> restore the original `user` property state before leaving the class.